src/App.css

```diff
@@ -1,41 +1,294 @@
-#root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
+@import url("https://fonts.googleapis.com/css2?family=Nunito:wght@200;400;700&display=swap");
```

Comment:
We should not write all of our CSS in one file. Please extract the component specific styles to their respective files.